webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -475,6 +470,111 @@ func (v validator) validateReadinessProbe(ctx *context.WebhookRequestContext, vm
 	return allErrs
 }
 
+func (v validator) validatePowerStateOnCreate(
+	ctx *context.WebhookRequestContext,
+	newVM *vmopv1.VirtualMachine) field.ErrorList {
+
+	var (
+		allErrs       field.ErrorList
+		newPowerState = newVM.Spec.PowerState
+	)
+
+	switch newPowerState {
+	case vmopv1.VirtualMachineGuestPoweredOff, vmopv1.VirtualMachineSuspended:
```

Comment:
Should this also be part of the API commentary on the `powerState` field that a VM can only be created in powered on and powered off state (self explanatory, but still)?